Open Badges 3.0 Proposal

kayaelle opened the pull request:
This is a proposal to align the IMS Global Open Badges Specification to the conventions of the W3C Verifiable Credentials Data Model for the use cases of Defined Achievement Claim and a Skill Claim. The credentials that would be produced under this proposal could easily be bundled into Comprehensive Learner Records and Verifiable Presentations. Portability and learner data privacy may be improved by expanding the usage of cryptographic proofs/signatures, because this format will be compatible with a growing array of proof schemas that are developed for the Verifiable Credentials Data Model.

Commit:
Draft of OBv3 proposal

ottonomy committed:
Merge pull request #1 from kayaelle/develop

Draft of OBv3 proposal

ottonomy committed:
Add more info to ob3 readme

Commit:
Added substantial content to specification.md

ottonomy committed:
Merge pull request #2 from kayaelle/feature/ob3

Added substantial content to specification.md

ottonomy committed:
Add clarity to 'What is the problem this solves for?' section

ottonomy committed:
Expand digital wallets section

ottonomy committed:
add section about DIDs and explanation of structural change

ottonomy committed:
merge upstream changes

ottonomy committed:
correct image syntax

ottonomy committed:
Add detail about skill assertions, small cleanups

ottonomy committed:
fix list item

Commit:
MInor content adjustments. Merging of duplicate content regarding 'Learner Data Privacy and Trustworthiness'

ottonomy committed:
Merge pull request #3 from kayaelle/feature/ob3

Minor content adjustments. Merging of duplicate content regarding 'Le…

Commit:
Edits from initial feedback

ottonomy committed:
Merge pull request #4 from kayaelle/feature/ob3

Edits from initial feedback

Commit:
Fixed typos. Moved example captions above

Commit:
Including Phil's feedback

Commit:
Some minor clean-up

Commit:
Adding introduction content, use cases, and terms

Commit:
Updating abstract and README

Commit:
Adding abstract & introduction to list of documents

Commit:
Minor content changes

Commit:
Sorting content. Edits

ottonomy committed:
Merge pull request #5 from kayaelle/feature/ob3

Updates to Proposal on content

ottonomy committed:
Spellcheck

Commit:
Adding contributors and fixing code examples

Commit:
Merge remote-tracking branch 'upstream/feature/ob3' into feature/ob3

Merging in from upstream.

Commit:
Fixing examples

ottonomy committed:
Merge pull request #6 from kayaelle/feature/ob3

Fixing Examples & Adding Contributors

Commit:
Fixed some formatting and figures

ottonomy committed:
Merge pull request #7 from kayaelle/feature/ob3

Fixed some formatting and figures

kayaelle commented:
[Open Badges 3.0 3.0 IMS Base Document.pdf](https://github.com/IMSGlobal/openbadges-specification/files/6779477/Open.Badges.3.0.3.0.IMS.Base.Document.pdf)

Commit:
Removing compact URI referencing property schema:hasCredential and replacing with just the property name, hasCredential

bracken commented on proposals/OBv3p0/respec-support/specification-wrapper.html:
```diff
@@ -0,0 +1,49 @@
+        specNature: "normative", // spec nature is "normative" or "informative"
+        specType: "proposal", // spec type is "main", "cert", "impl", "errata" or other agreed upon string
+        specVersion: "3.0",
+        specStatus: "IMS Base Document", // specStatus: IMS Base Document, IMS Candidate Final (Public) or IMS Final Release
```
Can you change `specStatus` to "Proposal" for now, we don't have a status for this yet so it won't look as nice, but Base Document isn't appropriate for proposals like this if PDFs and documents will be generated. 

We'll make the default template reflect this too so future proposals are already marked appropriately.

kayaelle replied:
Will do.

kayaelle replied:
All set

bracken replied:
Thanks!

Commit:
Fixing typo in third user story

Commit:
Changing specStatus to 'Proposal'

Commit:
Adding Respec generated proposal pdf

tkendal commented:
The alignment of OBs and VCs would represent a major step forward in open recognition. Thank you @kayaelle @ottonomy et.al. for your efforts here. More than happy to support this proposal!

dajbelshaw commented:
Making a note here of the comment I made in the W3C VC WG today about licensing of metadata in the context of 'Badge Creator'. I can go into more details if required, but it's been discussed in the Open Badges community for a while :)

longpd commented:
> Making a note here of the comment I made in the W3C VC WG today about (Creative Commons) licensing of metadata in the context of 'Badge Creator'. I can go into more details if required, but it's been discussed in the Open Badges community for a while :)

Doug: that's a good call out. The RSDs to have an explicit CC BY 4.0 International License associated with their use. But if I understand your point, you're suggesting something similar might be useful to distinguish the assertion as made by the issuer as distinct from the other aspects of the Badge as VC.  it probably would be useful for you to expand on that just a bit here.

dajbelshaw commented:
Thanks Phillip! (for those following along I had to look up [Rich Skill Descriptor = RSD](https://rsd.osmt.dev/)) My understanding is that RSDs are controlled vocabularies for skill descriptors? 

I'm thinking about a world in which badges (and badge systems/pathways) are designed by associations, trade bodies, or organisations other than those that issue them. In such a world, we'd presumably want to ensure that particularly well-constructed (or standardised) metadata could be re-used. 

In my experience, Creative Commons licenses work really well for this, but there may be other options - and use cases?

jischr commented on proposals/OBv3p0/specification.md:
```diff
@@ -0,0 +1,374 @@
+
+* _Issuer & Credential Subject Identities_: With VCs, [issuer](https://www.w3.org/TR/vc-data-model/#issuer) and [credentialSubject](https://www.w3.org/TR/vc-data-model/#credential-subject) id properties must be a URI. This could be an HTTP-based URL or a [Decentralized Identifier](https://www.w3.org/TR/did-use-cases/#intro).
+
+* _credentialSubject.hasCredential_: The hasCredential property from schema.org is intended to contain the BadgeClass properties including name, description, criteria, etc. hasCredential.id should contain the canonical url for the BadgeClass. hasCredential.type identifies the properties as belonging to the BadgeClass.
```
What is the idea behind hasCredential? It's a bit confusing to me since the Badge and VC would now be "one" - wouldn't all of the 3.0 Badges have credentials?

ottonomy replied:
The Verifiable Credentials Data Model offers a mechanism for an issuer to make "claims" about a credential subject. The whole package is a "Verifiable Credential", and each of the claims form the "substance" of what the issuer is describing about the subject. In comparison to VCs that make claims about your birth date `"birthDate": "2000-01-01"`, educational claims are often about the "credentials" you have received. It is a bit of overloading of the term, and the selection of exactly which term to use to form this relationship is something that the IMS OB Workgroup will dive into if we decide to move forward with integrating OB and VC. In OB 2.0 the relevant term is `"badge"` which translates to English as "this assertion claims that the recipient *has* this badge, in that they have met its criteria, and it has been conferred to them". We could stick with that term, but this is an opportunity to discuss if `hasCredential` from schema.org or another more generic `hasAchievement` term etc would be more fitting.

jischr commented on proposals/OBv3p0/specification.md:
```diff
@@ -0,0 +1,374 @@
+      "issuanceDate": "2010-01-01T19:23:24Z",
+      "credentialSubject": {
+        "id": "did:example:learner",
+        "hasCredential": {
```
does this imply a one-to-one relationship between badge and credential? would we want this to be an array?

on the contrary, what about a badge that requires multiple credentials to be satisfied?

ottonomy replied:
Like, "credential", "badge" is also an overloaded term. Sometimes it refers to the achievement definition (a badge that many learners might earn) and sometimes it refers to the assertion (the badge that Alice earned). 

The power of Open Badges often comes from the fact that they are granular and composable claims, each about one badge that one learner has earned. Allowing this to be an array instead of forming the array at a higher level (like a CLR) would undercut that composability and make it only possible for a recipient to share a bundle of all their badges that were issued together, or nothing at all. It probably doesn't make sense at this point to change the scope of what an Open Badges Assertion can claim in terms of the badge earned when we already have a great complementary concept of the CLR that can do that work (as well as introducing the possibility of using Verifiable Presentations as a mechanism to compose multiple credentials into one package that is greater than the sum of its parts.

ottonomy replied:
> what about a badge that requires multiple credentials to be satisfied?

This use case is very important to me, we have software in production that does this, and we would love to better make some of this data interoperable at the machine-readable level. To me this sounds like a use case for "Pathways", but perhaps there are some components that could be considered in badges beyond the human-readable criteria structure that allows people to express this information today. 

IMS is about to kick off a task force on Learning Pathways as part of the CASE work, that is also intended to be compatible with Open Badges and CLR as these specs all evolve together. The pathways task force kickoff registration is available [here](https://imsglobal.zoom.us/meeting/register/tJAtc-GrqzsrGdU2s66zVR64Xb4jyR718BdB) as part of the [Tech Congress agenda](https://www.imsglobal.org/event/ims-technical-congress-2021) for August 4th. 


However we name these concepts going forward and charter specifications and workgroups to cover them, these ideas should all fit together neatly. Here's the IMS groups covering this type of scope:
- Skill and Competency Definitions (Competencies and Standards Exchange  - CASE)
- Credential definitions and Assertions (Open Badges)
- Transcripts and Bundles of Credentials (Comprehensive Learner Record - CLR)
- Learning Pathways, both prescriptive and descriptive (CASE Pathways Task Force)

roverwolf commented on proposals/OBv3p0/specification.md:
```diff
@@ -0,0 +1,374 @@
+
+* _credentialSubject.evidence_: With OpenBadges up to 2.0, the evidence has been an assertion property. This proposal suggests that evidence should be included in the credentialSubject object so that the evidence is related to the claim, not the verification of the claim.
+
+* _credentialSubject.resultDescription_: This property is taken from the CLR and may follow the same recommendations. As with evidence, this property should be included in the credentialSubject object.
```
Pulling in the _resultDescriptions_ from CLR Achievements is probably not very useful without also pulling in the _results_ from CLR Assertions.

I would also think the field would make more sense under the _hasCredential_ as it is part of the definition of a credential/achievement/BadgeClass and not related to the subject.

ottonomy replied:
@roverwolf Yes this is correct. _credentialSubject.result_ would be the right field to reference here. `resultDescription` should be in the Achievement/BadgeClass class.

OR13 reviewed (approved):
These changes look great!

roverwolf commented on proposals/OBv3p0/specification.md:
```diff
@@ -0,0 +1,374 @@
+
+Verification of control of identifiers is an important concept within any type of digital credential, both with respect to the issuer and the subject (recipient) of the credential. For issuers, Open Badges has relied on its own bespoke rules for determining whether a hosted Assertion URL or cryptographic key URL is associated with an issuer profile identified by a particular URL. URLs used for re ...
+
+DIDs typically offer cryptographic proof of control, based on authorized keys or other verification methods expressed in the associated DID Document. While these protocols are not broadly implemented across domains today, the structure provides a forward-looking flexible and extensible mechanism to build the types of protocols needed to connect credentials back to the identities of their issuers  ...
```
I think it is important to note that a user's proof of control of a recipient DID does not imply that the DID represents that user, such as in cases of youth where the DID may be controlled by their guardian.  Methods of identifying a specific person from a DID will probably need to be specified (or from other information embedded in the badge).

justinpitcher commented:
> `Revoked Credentials: VCs have a credentialStatus object`

Are we intended to re-use the existing [Credential Status List](https://w3c-ccg.github.io/vc-csl2017/) type or define a new one that partially/wholly replicates the OB 2.0 revocation list? And could this be an opportunity to expand the concept to BadgeClasses themselves. At various times we've bounced around the idea of listing badge classes which have been retired/deleted/culled.

GregNadeau commented:
I oppose the proposal for OB to continue forked from CLR.

A significant portion of achievement assertions are published by the issuer or shared by the holder in sets with important relationships to each other, not only as single claim assertions.   Both the simple/single claim and more complex sets of claims should operate with similar VC schemas and protocols, both should be represented in IMS’s next digital credential standards

I recommend that the proposal to continue forking OB from CLR should be merged into a charter to bring together CASE, OB, CLR, and HR Open into a CLER digital credential standard that will meet diverse market needs.  

As agreed at the end of the OB Task Force, we will take up this matter at the CLR Task Force this week.

longpd commented:
Many individuals engage in learning activities the successful completion of which result in the issuance of single assertion badges. The rich OBv2.1 data structure organizes the description of the work done, rubric used, possible alignment to a schema.org assessment methods, and the possible inclusion of an evidence array that points to representative samples of the individuals work for which the badge was issued. These have been awarded by issuers for years with this narrow focus, and we expect will continue to be going forward. 

The VC data model accommodates a range of payloads, educational as well as myriad others (age credentials, naturalization status, drivers licenses, etc.). It’s the holder's choice as to how to aggregate them to present to a relying party. The OBv3 proposal is focused on one issuer of one claim about one recipient, with a robust, flexible, and future proof solution that can easily be integrated with the set-of-multiple credentials use cases familiar
to Comprehensive Learner Record. But there has not been any intention in the design of either OBv2 or the proposed OBv3 VC compatible badge to represent them as a coordinated set beyond the holders decision to send them together. 

The work on OBv3 by the badge work group should share the design decisions and methods that are developed with a parallel CLR VC working group. The relative simplicity of the OBv3 proposal can offer some suggestions as to how such a workgroup looking toward harmonizing the CLR with VCs might proceed. But it is well understood that the complexity of the CLR will require broader input and discussions as it iterates toward a recommended VC-CLR roadmap.

Finally, the IMSGLOBAL OB workgroup has spent 4+ years on open badges, inheriting significant prior work on open badges from the Mozilla Foundation badge initiative. The characterization that the OBv3 proposal “continue[s] forking OB from CLR“ is both historically and factually misleading. While I’m not an expert on the early decisions that led to the pathways pursued by the CLR workgroup, the OBv3 proposal was proposed to leverage the developments in the VC ecosystem, and opportunities therein. It is hoped that it can inform the CLR-to-VC pathway work enabling the CLR to become a full-fledged component of the growing VC ecosystem.

GregNadeau commented:
Phil makes many good points which I agree with.  I believe the crux of any disagreement may be whether we tightly or loosely coordinate the integration of VC (and HR Open) with OB and CLR.  I believe we should closely align and will make that case on Thursday.

More detailed responses follow:

1. Many individuals engage in learning activities the successful completion of which result in the issuance of single assertion badges. 
GN=> Agreed

2. The rich OBv2.1 data structure organizes the description of the work done, rubric used, possible alignment to a schema.org assessment methods, and the possible inclusion of an evidence array that points to representative samples of the individuals work for which the badge was issued. These have been awarded by issuers for years with this narrow focus, and we expect will continue to be going forward.
GN=>Disagree.  As acknowledged here, narrow focus badges have been issued for years, complex derivatives such as hypothesized above are still unproven in production.

3. The VC data model accommodates a range of payloads, educational as well as myriad others (age credentials, naturalization status, drivers licenses, etc.). It’s the holder's choice as to how to aggregate them to present to a relying party. 
GN=>Agreed. A verifiable CLR set is packaged by the issuer when the context is like a transcript with immutable parts.

4. The OBv3 proposal is focused on one issuer of one claim about one recipient, with a robust, flexible, and future proof solution that can easily be integrated with the set-of-multiple credentials use cases familiar to Comprehensive Learner Record. 
GN=>  Disagree.  Let’s do it now, not 2 years from now.  As there would be significant opposition to this proposal going forward, the fastest path will be to work together and jointly public a VC + OB/CLR with possible HR Open extensions.

5. But there has not been any intention in the design of either OBv2 or the proposed OBv3 VC compatible badge to represent them as a coordinated set beyond the holders decision to send them together.
GN=> Agreed.  That is insufficient for a significant set of scenarios (e.g. transcripts)

6. The work on OBv3 by the badge work group should share the design decisions and methods that are developed with a parallel CLR VC working group. 
GN=> Disagree.  These efforts should be more tightly integrated.

7. The relative simplicity of the OBv3 proposal can offer some suggestions as to how such a workgroup looking toward harmonizing the CLR with VCs might proceed. But it is well understood that the complexity of the CLR will require broader input and discussions as it iterates toward a recommended VC-CLR roadmap.
GN=> Maybe.  Or, it could take us down a dead end that does not support the range of scenarios needed.

8. Finally, the IMSGLOBAL OB workgroup has spent 4+ years on open badges, inheriting significant prior work on open badges from the Mozilla Foundation badge initiative. The characterization that the OBv3 proposal “continue[s] forking OB from CLR“ is both historically and factually misleading. While I’m not an expert on the early decisions that led to the pathways pursued by the CLR workgroup, the OBv3 proposal was proposed to leverage the developments in the VC ecosystem, and opportunities therein. 
GN=>  This is the crux of the misunderstanding.  I co-chaired the CLR task force and directed the initial close alignment with OB2.  I can say with authority that our intention then was to build CLR off the foundation of OB2.  CLR 1.0 is, effectively OB 3.0.  The intention of the Digital Credential steering committee was always to harmonize future standards with IMS.  This proposal would not do that.

9.  It is hoped that it can inform the CLR-to-VC pathway work enabling the CLR to become a full-fledged component of the growing VC ecosystem.
GN=>  And the time to do that is now.

kayaelle commented:
The Open Badges 3.0 proposal is intended to increase the value proposition of Open Badges as a type of digital credential for skills and achievements that happen anywhere at any time. In the past few years we've seen an uptick in institutions issuing Open Badges and over the past decade, we know that digital credentials presented as single achievements have been issued in various contexts outside of institutions including peer-to-peer recognition. 

Starting in 2011, the Open Badges community introduced and fought for experiential and skills-based recognition having value beyond or alongside degrees. A primary point of Open Badges has always been to increase social equity by making it possible for those without the means to pursue careers that they are qualified for, even when they can't afford to complete a degree. Currently, this is even more critical as organizations like the T3 Innovation Network, and Open Skills Network, and the Walmart Foundation have vocally demonstrated support for skills-based learning. Employers are seeking those who have the skills needed for their jobs, even if they don't have degrees. 

Degrees are still valued and digital standards like the CLR that represent entire transcripts serve institutions well by representing those complex education histories. But Open Badges has earned its own place in this ecosystem by serving a broader population and as a lighter-weight yet data-rich solution for digital recognition. I'm certain that IMS Global, its contributing members, especially the Open Badges project group members, can see that the initiatives propagated by Open Badges have taken hold. It's been a long road for this movement and it would be unfortunate to see adoption of skills-based hiring get stalled by phasing out the very digital credential that has propagated it at the very moment when it's gaining traction globally. 

This 3.0 proposal prioritized Open Badges alignment with VCs first because Open Badges is well-known and well-suited to be aligned with the VC Data Model. The Open Badges community wrote some of the very first use cases for VCs. It's true that CLRs can also be aligned with VCs and it's unfortunate that this wasn't considered while the CLR was being developed. Perhaps this is because CLRs are much more complex models and it will take some time to get them aligned with VCs and verifiable presentations. Open Badges 3.0 can be ready for use by pilots in the fall because much of the preparations have been done over the past three years. This work can inform a CLR update that can also be in alignment with VCs. In fact, that work can and should begin in parallel as its own work item. 

Please note that @GregNadeau has made a couple of misleading statements in his position that I would like to clarify: 

1. Open Badges was not forked from the CLR. Open Badges is a significantly more mature and much more widely used standard celebrating its 10 year anniversary this year. Open Badges has a robust validator and a standardized API to encourage badge interoperability.  CLR v1 just launched last year with a final release approved in January 2021 . 

2. The agenda for the meeting planned for this Thursday (Aug 5) during the CLR project group regular meeting is to invite Open Badges IMS members to continue the discussion about this proposal and how it can positively affect how the CLR can also be better aligned with VCs. @GregNadeau's opposition statement is an overreach. If Public Consulting Group is interested in submitting a proposal to merge the standards, they should follow the same template provided by IMS Global and used by Concentric Sky to document it in detail so that it can be commented on and considered by IMS Members. 

As someone who has been personally involved in the development and support of every version of Open Badges, I can tell you that Open Badges and CLR have very little in common. If the intention was to phase out Open Badges, this was not common knowledge and it is not evident in the architecture of the CLR or its documentation. There’s an opportunity to align these standards in powerful ways and there’s no reason why they can’t continue to co-exist. The Open Badges 3.0 proposal has the capacity to catapult Open Badges and CLRs. Let’s make that happen.

GregNadeau commented:
HI Kayaelle -  As a point of fact when I joined CLR TF leadership I literally copied OB2 into a Google Doc and changed the name to CLR to create the new base document.  CLR is 100% based on OB.  We spent two years refining it, making it more extensible, and making the terminology more academic-friendly.  I look forward to talking with you and others more on Thursday about how we can work together going forward.

dajbelshaw commented:
As someone who has been involved with Open Badges since the very beginning (I was on the original Mozilla team) I find this discussion of the CLR as somehow the apotheosis of Open Badges very concerning.

As IMS is a closed shop, it hasn't been possible for me to be involved in development of the standard over the last four years. However, wherever I've been able to support the move to make Open Badges more closely aligned with Verifiable Credentials, I have done.

From what I understand, the attempt to highjack the Open Badges standard by fans of the CLR seems a very US-centric and a somewhat reactionary move.

For what its worth, and on behalf of the Open Badges community not represented by IMS, I'd like to express my support for continuing the evolution of Open Badges towards Verifiable Credentials. Thank you to Kerri, Nate and others for keeping the original vision of a learner-centric open standard alive!

-------- Original Message --------
On 2 Aug 2021, 00:12, kayaelle wrote:

> The Open Badges 3.0 proposal is intended to increase the value proposition of Open Badges as a type of digital credential for skills and achievements that happen anywhere at any time. In the past few years we've seen an uptick in institutions issuing Open Badges and over the past decade, we know that digital credentials presented as single achievements have been issued in various contexts outside of institutions including peer-to-peer recognition.
>
> Starting in 2011, the Open Badges community introduced and fought for experiential and skills-based recognition having value beyond or alongside degrees. A primary point of Open Badges has always been to increase social equity by making it possible for those without the means to pursue careers that they are qualified for, even when they can't afford to complete a degree. Currently, this is even more critical as organizations like the T3 Innovation Network, and Open Skills Network, and the Walmart Foundation have vocally demonstrated support for skills-based learning. Employers are seeking those who have the skills needed for their jobs, even if they don't have degrees.
>
> Degrees are still valued and digital standards like the CLR that represent entire transcripts serve institutions well by representing those complex education histories. But Open Badges has earned its own place in this ecosystem by serving a broader population and as a lighter-weight yet data-rich solution for digital recognition. I'm certain that IMS Global, its contributing members, especially the Open Badges project group members, can see that the initiatives propagated by Open Badges have taken hold. It's been a long road for this movement and it would be unfortunate to see adoption of skills-based hiring get stalled by phasing out the very digital credential that has propagated it at the very moment when it's gaining traction globally.
>
> This 3.0 proposal prioritized Open Badges alignment with VCs first because Open Badges is well-known and well-suited to be aligned with the VC Data Model. The Open Badges community wrote some of the very first use cases for VCs. It's true that CLRs can also be aligned with VCs and it's unfortunate that this wasn't considered while the CLR was being developed. Perhaps this is because CLRs are much more complex models and it will take some time to get them aligned with VCs and verifiable presentations. Open Badges 3.0 can be ready for use by pilots in the fall because much of the preparations have been done over the past three years. This work can inform a CLR update that can also be in alignment with VCs. In fact, that work can and should begin in parallel as its own work item.
>
> Please note that ***@***.***(https://github.com/GregNadeau) has made a couple of misleading statements in his position that I would like to clarify:
>
> -
>
> Open Badges was not forked from the CLR. Open Badges is a significantly more mature and much more widely used standard celebrating its 10 year anniversary this year. Open Badges has a robust validator and a standardized API to encourage badge interoperability. CLR v1 just launched last year with a final release approved in January 2021 .
>
> -
>
> The agenda for the meeting planned for this Thursday (Aug 5) during the CLR project group regular meeting is to invite Open Badges IMS members to continue the discussion about this proposal and how it can positively affect how the CLR can also be better aligned with VCs. ***@***.***(https://github.com/GregNadeau)'s opposition statement is an overreach. If Public Consulting Group is interested in submitting a proposal to merge the standards, they should follow the same template provided by IMS Global and used by Concentric Sky to document it in detail so that it can be commented on and considered by IMS Members.
>
> As someone who has been personally involved in the development and support of every version of Open Badges, I can tell you that Open Badges and CLR have very little in common. If the intention was to phase out Open Badges, this was not common knowledge and it is not evident in the architecture of the CLR or its documentation. There’s an opportunity to align these standards in powerful ways and there’s no reason why they can’t continue to co-exist. The Open Badges 3.0 proposal has the capacity to catapult Open Badges and CLRs. Let’s make that happen.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, [view it on GitHub](https://github.com/IMSGlobal/openbadges-specification/pull/303#issuecomment-890603050), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AASRD4QD2T3DJS7VXMCD3QLT2XIFTANCNFSM475STOWQ).

kayaelle commented:
> HI Kayaelle - As a point of fact when I joined CLR TF leadership I literally copied OB2 into a Google Doc and changed the name to CLR to create the new base document. CLR is 100% based on OB. We spent two years refining it, making it more extensible, and making the terminology more academic-friendly. I look forward to talking with you and others more on Thursday about how we can work together going forward.

@GregNadeau - I think your point demonstrates why the CLR is not like Open Badges and why they should remain separate. You may have started with Open Badges, and CLRs may even contain Open Badges, but they are very different specifications. Open Badges has always been more than a data spec. You strove to make "the terminology more academic-friendly" but then pulled away from the point of Open Badges and away from the community that has served it and used it so well over the years. Open Badges can serve academia but it is intended to accommodate learning in any scenario including workplace training, and autodidactism which are not the objectives of academia. 

From a personal perspective as an example, I'm a highly educated individual but prior to going back to academia, I worked as a software developer for most of my career. The skills I gathered prior to finishing my degrees are the ones that are most applicable to my day-to-day work still. They are the skills that can be Open Badge-able by me, by my previous employers, and by my colleagues. Open Badges didn't exist as I was teaching myself to code and it was intensely challenging to prove to potential employers that I could do the job without a degree. Open Badges has filled that gap for many over the past decade. I've personally witnessed its impact and have been part of a community that continues to find purposeful implementations of Open Badges including at major corporation and in governments around the world. 

My hope is the the Open Badges 3.0 proposal is seen as a way to strengthen its ethos by aligning with Verifiable Credentials and [Self-sovereign Identity](https://github.com/WebOfTrustInfo/self-sovereign-identity/blob/master/self-sovereign-identity-principles.md). As stated above by both Phil and me, the intention is to inform future versions of the CLR with this update. This could lead to more harmonization of the two specs in the future but there's significant work ahead to make that a consideration. So why hold back the spec that's ready to move in this direction now?

TheGrumbledook commented:
To be open, there is little I can add to the discussion around the technicalities of the specifications and differences between OB v3 and CLR, but as an end-user and proponent for modular and verifiable certificates, I always look for an open standard to work with that I can encourage EdTech vendors to build into their approaches. 
As a privacy professional, transparency is essential and fairness are essential. I will always ask an open project to let me know what privacy impact assessments have been done during development, what peer review has there been and where has privacy by design and by default been considered (security by design and by default seems to be a given on most things nowadays but that is my next tier of questions).
I'd like both groups to explain their positions on this as I am keen to have a better understanding.

teresamac commented:
I find it distressing that an innovation which currently offers so much to learners and practitioners alike due to being founded on openness and transparency could abandon such values. The growth in popularity of open badges is largely down to work done openly to promote this form of recognition in such projects as virtual exchange. This work has gained attention but would no longer be supported if badge awarding came adrift from these essential values.

dpresant commented:
As a current member of IMS, a member of the Open Badges community for 10 years and a firm advocate of the **open recognition** principles that Mozilla Foundation invented open badges to support, I oppose the merging of Open Badges and CLR. 

I believe the respective communities and the recognition ecosystem would be better served by keeping these two standards autonomous and interoperable. More robust, more flexible, more global.

szerge commented:
I'm joining the conversation following an invitation from @dajbelshaw to comment on the intention to supersede OB specifications with that of the Comprehensive Learner Record. I understand that there are people who believe that a CLR is a better OB and they might be right—or wrong, but that's not my point. 

We should agree that the people who are best placed (and legitimate) to decide whether or not OBs should be superseded by CLRs are those working with Open Badges _today_, not those who would like to work with something _like Open Badges_ if they were CLRs (which according to @GregNadeau is a fork from the original OB spec).

The main problem I see with a _Learner Record_ (comprehensive or not) is that it only covers a subset of what Open Badges are being used to. Open Badges are used to _recognise_ the members of a community of practice ("I belong to..."), they are also used to _document_ a practice that can be therefore _recognised_ (endorsed) by other members of the community of practice and those interacting with that community.  They are also used to _recognise_ (make visible) not just just people but also practices, knowledge, skills, commitments, projects, etc.

A _Learner Record_ conveys the idea of "something that happened in the past" and was recorded. Open Badges are used to create identities as projection into the future (I claim the "Doctor" badge to show to the world that I want to become a doctor and use it to guide my education and social engagement). Should we create a special type of _Learner Record_ that wouldn't be a record of past actions but _projection_ into the future? No need for that: we already have Open Badges!

So, while I understand that CLR proponents believe that a CLR is a better OB, _for what they want to do with them_, they should make the effort to understand why a number of Open Badge practitioners believe that OBs are better CLRs to do what we are doing _today_.

wizzwizz4 commented:
I don't see any reason CLR shouldn't be an extension to Open Badges, but specialising the entirety of OBv3 in this way would break the use-cases that got me so excited about Open Badges in the first place. We've barely begun to unlock the potential of computer technology in education; we need more flexibility in what constitutes education, not a better way of specifying _existing_ kinds of education.

ColinSmythe commented:
Can we please not circulate incorrect information. The blog post be @dajbelshaw stating that a Member vote will take place this week is INCORRECT. We are in a phase of broad discussion and opinion gathering. Formal IMS Member votes take place at ONLY two points in a specification development process (Charter acceptance and Final Release acceptance). A Charter takes take several weeks to write before going to the IMS TAB. The TAB voting process typically takes a minimum of two weeks. It is irresponsible to distort these important discussions with factually incorrect statements.

dajbelshaw commented:
> I expect @dajbelshaw to correct the blog.

Sure. Just so I get it right this time, how long do community members and real-world users have to give their input, and where would you like them to share that feedback? :slightly_smiling_face:

ColinSmythe commented:
Thank you.
No deadline has been set. I would expect several more weeks of discussion and reflection. 
For the time being feedback should be to this GitHub repo.

misterppqx commented:
Open Badges are not just micro credentials.  It's more for us especially in France a tool to empower individuals and communities. With OBs we do much more than just displaying micro things. We use them as connectors, we use them to tell who we are, what we stand for, what we learn,  what we can teach to others, what we want to be..and who recognizes us for that.
Thanks to endorsements,  we can support the people's power to recognize others and just like wikipedia did with opening knowledge,  open the recognition. 
Ten years after we would like to make open badges more simple to use and not a labyrinthine system. As the coanimator of the french speaking branch of the openrecognition alliance , I oppose the merge of CLR with open badges. 
Merci.

sgrrant commented:
@GregNadeau I can see where you're coming from, and can even understand that for you/your clients "OB2. CLR 1.0 is, effectively OB 3.0."

As an innovation, badges are something of an inkblot test. We see their potential through the lens of what we know and value. 

There is fierce resistance to the proposed harmonization of the data standards because CLR 1.0 is a narrower view or solution than what inspired such excitement about OB 1.0 in 2011. We are still in the early stages of building a badge-issuing culture, much less a consumption culture. 

CLR 1.0 may be an elegant and even wonderful solution that solves marketplace issues for IMS Global's paying members and/or your clients. However, had Mozilla launched CLR 1.0 instead of OB 1.0 back in 2011 I suspect the reception would have been crickets, relatively speaking. Badges proved to inspire people in a big tent way that doesn't often happen. 

It has been a good exercise for me to reflect on why CLR 1.0 does not inspire the same response as OB 1.0, and ask what arguments or forces are in play to make the case that CLR/OB should be harmonized. Maybe you can weigh in on that so we can share your thinking more widely with our respective communities and the wider open badge community.

ColinSmythe commented:
@sgrrant IMS Specifications are open and free to use for ALL. From our analysis there are anywhere between 3 and 10 times more vendors implementing/using specifications than there are IMS Members. IMS specifications have considerably more appeal the for just "IMS Global's paying members".

ColinSmythe commented:
@sgrrant IMS Specifications are open and free to use for ALL. From our analysis there are anywhere between 3 and 10 times more vendors implementing/using specifications than there are IMS Members. IMS specifications have considerably more appeal the for just "IMS Global's paying members".

justinpitcher commented:
While I can envision how a viewpoint of, "CLR 1.0 is, effectively OB 3.0" came to be, I can say that from my perspective this is not a baseline belief for the standard. I was a participant in both the extended transcript group and the early definitive meetings of CLR and this characterization was never part of any conversation I participated in.

Compatible, ecosystem-oriented standard? Yes. Subsuming standard? Absolutely not.

debeverhart commented:
@ColinSmythe -isn't the wide range of community engagement and use of Open Badges precisely why this specification needs to evolve with unfettered access to open, global, free community involvement? As a very early Open Badges work group contributor and ongoing advocate, I know first hand the benefits of global engagement and the amazing talent and creativity empowered when we remove any and all barriers to the free flow of ideas

Upperholme commented:
Thanks to @dajbelshaw for pointing me to this. Like other commenters here I see the huge value and potential that Open Badges have beyond the confines of academia and the formal education industry. Mr Nadeau and his colleagues seem to have very narrow view of this issue. His suggested approach strikes me as the tail wagging the dog, and would be a huge loss if it came to pass. As others have already said, the potential for Open Badges has yet to be realised. Painting it into a corner to serve the interests of just one constituency of users strikes me as counter-productive for just about everyone involved.

Commit:
Introduction.md

- 2nd paragraph Design Goals and Rationale
- Mapping Skills use case - changed persona to a non-binary gender
- Terms: Added Achievement, Assertion, Result

Specification.md

- Aligning Open Badges and CLR, Common Assertion Model, and Common Achievement Model to Benefits and Opportunities
- Changed hasCr edential to achievement, added credentialSubject.result to Changes in the Data Model
- Updated examples to change hasCredential to achievement and correct result/resultDescriptions

ottonomy committed:
update link text to dtResult class

ottonomy committed:
Tweak language and update included PDF to latest version

ottonomy committed:
Fix markdown error and heading level error, regenerate PDF

kayaelle commented:
Summary of changes made today (See attached revised PDF):

Introduction.md

- 2nd paragraph Design Goals and Rationale
- Mapping Skills use case - changed persona to a non-binary gender
- Terms: Added Achievement, Assertion, Result

Specification.md

- Aligning Open Badges and CLR with Common Assertion and Achievement Models in Benefits and Opportunities section
- Changed hasCredential to achievement, added credentialSubject.result to Changes in the Data Model section
- Updated examples to change hasCredential to achievement and correct result/resultDescriptions

[Proposal-Open-Badges-3.0.pdf](https://github.com/IMSGlobal/openbadges-specification/files/6971586/Proposal-Open-Badges-3.0.pdf)

Commit:
Update prettier version of same proposal PDF

Commit:
Fixing images in proposal PDF
